hummingbot/connector/derivative/dydx_perpetual/dydx_perpetual_derivative.py

```diff
@@ -135,60 +115,63 @@ def logger(cls) -> HummingbotLogger:
             s_logger = logging.getLogger(__name__)
         return s_logger
 
-    def __init__(self,
-                 dydx_perpetual_api_key: str,
-                 dydx_perpetual_api_secret: str,
-                 dydx_perpetual_passphrase: str,
-                 dydx_perpetual_account_number: int,
-                 dydx_perpetual_ethereum_address: str,
-                 dydx_perpetual_stark_private_key: str,
-                 poll_interval: Optional[float] = None,
-                 trading_pairs: Optional[List[str]] = None,
-                 trading_required: bool = True):
+    def __init__(
+        self,
+        dydx_perpetual_api_key: str,
+        dydx_perpetual_api_secret: str,
+        dydx_perpetual_passphrase: str,
+        dydx_perpetual_account_number: int,
+        dydx_perpetual_ethereum_address: str,
+        dydx_perpetual_stark_private_key: str,
+        trading_pairs: Optional[List[str]] = None,
+        trading_required: bool = True,
+    ):
 
         ExchangeBase.__init__(self)
         PerpetualTrading.__init__(self)
         self._real_time_balance_update = True
-        self.API_REST_ENDPOINT = MAINNET_API_REST_ENDPOINT
-        self.WS_ENDPOINT = MAINNET_WS_ENDPOINT
+        self._shared_client = aiohttp.ClientSession()
```

Comment:
We should not use the aiohttp session directly. We have to use the new RESTAssistant (check GateIO exchange as an example)